### PR TITLE
C5s4 adamggg reset password

### PR DESCRIPTION
**Pull request by adamggg**
#354

Scenario Reviewer: @KareemAliRahman
Implementation Reviewer: @NourhanMohamed
UML Reviewer: @Khloud-Khalid
Documentation Reviewer: @zoheiry

### CODE REVIEW

File: arability/app/views/devise/passwords/new.html.erb [hunk 1]
```diff
@@ -1,12 +1,10 @@
-<h2>Forgot your password?</h2>
 
-<%= form_for(resource, :as => resource_name, :url => password_path(resource_name), :html => { :method => :post }) do |f| %>
-  <%= devise_error_messages! %>
-
-  <div><%= f.label :email %><br />
-  <%= f.email_field :email %></div>
-
-  <div><%= f.submit "Send me reset password instructions" %></div>
+<%= form_for(resource, :as => resource_name, :url => password_path(resource_name, :locale => I18n.locale), :html => { :method => :post, class: "form-signin", style: "min-width: 380px;" }) do |f| %>
```
Comment (NourhanMohamed) [inline arability/app/views/devise/passwords/new.html.erb]: remove all =>

File: arability/app/views/devise/passwords/edit.html.erb [hunk 2]
```diff
@@ -1,16 +1,12 @@
-<h2>Change your password</h2>
 
-<%= form_for(resource, :as => resource_name, :url => password_path(resource_name), :html => { :method => :put }) do |f| %>
-  <%= devise_error_messages! %>
+<%= form_for(resource, :as => resource_name, :url => password_path(resource_name, :locale => I18n.locale), :html => { :method => :put, class: "form-signin", style: "min-width: 380px;" }) do |f| %>
```
Comment (NourhanMohamed) [inline arability/app/views/devise/passwords/edit.html.erb]: remove all =>

File: arability/app/views/devise/mailer/reset_password_instructions.html.erb [hunk 3]
```diff
@@ -1,8 +1,11 @@
-<p>Hello <%= @resource.email %>!</p>
+<div style="<%= 'direction: rtl; font-size: 20px;' if I18n.locale == :ar %>">
```
Comment (NourhanMohamed) [inline arability/app/views/devise/mailer/reset_password_instructions.html.erb:1]: use double quotes

Comment (adamggg) [inline arability/app/views/devise/mailer/reset_password_instructions.html.erb:1]: For the ruby part of the code I am using double quotes, the inner part is just html that will get embedded escaping the characters will put more chars than actually decreasing the code. Wouldn't it?

**Comment by adamggg** [comment]
@NourhanMohamed , please review =)

File: arability/app/views/devise/mailer/reset_password_instructions.html.erb (hunk 3 above)
Comment (NourhanMohamed) [inline arability/app/views/devise/mailer/reset_password_instructions.html.erb:1]: oh I didn't see the outer surrounding quotes .. It's okay then =)

File: arability/config/routes.rb [hunk 4]
```diff
@@ -39,7 +39,8 @@
   scope "(:locale)", :locale => /en|ar/ do
 
     # required for routing by the devise module(gem)
-
+    # devise_for :gamers, :controllers => { :omniauth_callbacks => "omniauth_callbacks" }
```
Comment (NourhanMohamed) [inline arability/config/routes.rb:45]: remove comment

Comment (adamggg) [inline arability/config/routes.rb:45]: This is something that Amr has put up, not me :)

AG

Comment (adamggg) [inline arability/config/routes.rb:45]: @NourhanMohamed I accidentally deleted and put it back so it got blamed one me =)

Comment (NourhanMohamed) [inline arability/config/routes.rb:45]: lol I wasn't going to git blame after you :D

Comment (adamggg) [inline arability/config/routes.rb:45]: Do you still want me to remove it? =D

**Pull request merged**